test/modelManager/usermodel.test.js

```diff
@@ -0,0 +1,31 @@
+const { expect } = require('chai');
+const { LIMITHELPER, OFFSETHELPER } = require('../../constants');
+const User = require('../../lib/modelManagers/usermodel');
+
+describe('Unit test usermodel delete function', () => {
+  it('s should', async () => {
+    const res = await User.delete('846c6586-1a65-4829-8d77-9318c6bbf080');
+    expect(res).to.equal(0);
+  });
+});
+
+describe('Unit test usermodel create function', () => {
+  it('s should', async () => {
+    const res = await User.create('test@gmail.com', 'password', 'firstname', 'lastname');
+    expect(res).to.be.an('object');
+  });
+});
+
+describe('Unit test usermodel update', () => {
+  it('s should', async () => {
+    const res = await User.update('846c6586-1a65-4829-8d77-9318c6bbf080', 'id');
+    expect(res).to.be.an('array');
```

Comment:
The assertion does not fully cover the function. Please add more assertions

Reply:
done